### DISCUSSION TITLE
Can we use other models, say LLama or Gemini ?

### DISCUSSION THREAD

**Discussion by SwastikGorai** [microsoft/graphrag#436] [Q&A]
Can we use models hosted in cloud, or gemini api? Or can we use Ollama if we are hosting the model locally?

**Comment by Goodnight77** [comment]
i used gemma2 locally where i installed it with ollama and also used mixtral from groq api

**Reply by jess-99** [reply]
how did you edit the source code to use other models, also did you tried to use other vectorstores than azure?

**Reply by jaigouk** [reply]
change settings.yaml. In my case, I was able to use Claude 3 haiku https://github.com/microsoft/graphrag/issues/471#issuecomment-2227155764

**Comment by natoverse** [comment]
Have a look at #657

**Comment by duongkstn** [comment]
Try litelm, it worked in my case !
